hera_qm/ant_metrics.py

```diff
@@ -140,7 +140,8 @@ def calc_corr_stats(data_sum, data_diff=None, flags=None, time_alg=np.nanmean, f
             corr_stats[bl] = np.abs(time_alg(freq_alg(even * np.conj(odd), axis=1)))
     return corr_stats
 
-def corr_metric(corr_stats, xants=[], pols=['ee','nn']):
+
+def corr_metric(corr_stats, xants=[], pols=None):
```

Comment:
I had set the default for pols to be ['ee','nn'] because in general we only want the same-pols to go into this metric. I'm fine with reconfiguring how that default is implemented, but with your implementation all 4 pol combinations would go into the metric, which is suboptimal.

Reply:
I don't think that's true, because of how it's called within the AntennaMetrics class: https://github.com/HERA-Team/hera_qm/blob/5b1c640a78de31cba902eeeba2571f5f20bdcd94/hera_qm/ant_metrics.py#L461

Reply:
Good point, it's fine how it is then.